### ISSUE TITLE
Suggestion: Add blacklist or whitelist for pet bed

### ISSUE THREAD

**Issue by Star-Jirachi** [Alex-the-666/DomesticationInnovation#181]
Add blacklist or whitelist for pet bed, it can fix some bug

**Comment by donmor** [comment]
Agreed, bots in [this mod](https://www.curseforge.com/minecraft/mc-mods/reboot-lovelyrobot) respawns from their cores, but later they respawn again from the bed, causing duping happens.

**Comment by donmor** [comment]
Making a datapack prevents mobs from using beds, but it'll be better if there's an in-game blacklist.
